webapp/app/templates/components.html

```diff
@@ -152,12 +152,12 @@ <h5 class="mb-0">{{ card['heading'] }}</h5>
                 <div class="row collapsed" data-toggle="collapse"
                             data-target="#details-body-{{ details_id }}"
                             aria-expanded="false" aria-controls="collapseOne">
-                    <button class="btn btn-primary container" type="button">
+                    <button class="container" type="button">
```

Comment:
There is a lot more stuff in btn (e.g. animations, no text-selection) that we'll lose. I am not saying that we absolutely need them, but maybe we should think about that. However, we do not need btn-primary.

Reply:
Nice. Thanks for the comment, I re-added it. :)